src/engine/index.ts

```diff
@@ -235,6 +235,14 @@ class Engine implements RPCHandler {
     return new Set();
   }
 
+  async getCustodyEventByUser(fid: number): Promise<IDRegistryEvent | undefined> {
+    const signerSet = this._signers.get(fid);
+    if (signerSet) {
```

Comment:
teeny-tiny nit: can use ternary to make the code easier to read 

```suggestion
    return signerSet ? signerSet.getCustodyAddressEvent() : undefined;
```